src/System.Diagnostics.DiagnosticSource/src/Configurations.props

```diff
@@ -2,7 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      net45-Windows_NT;
       netstandard1.1;
```

Comment:
If you are removing the netstandard1.1 ref you should probably do the same for the src.